### ISSUE TITLE
customcontrols: hide in print view

### ISSUE THREAD

**Issue by jceb** [rajgoel/reveal.js-plugins#159]
Thank you for this great selection of plugins! I'd like to use the PDF export capability of revealjs and noticed that the customcontrols buttons show up in the export view. Could they be hidden? I played around with the `aria-status` class and it seems to work. However, I'm not sure if this is a good class to use when not in the print view.

**Comment by jceb** [comment]
Including the following snippet will hide the custom controls when printing in the browser:

```css
@media print {
  #customcontrols {
    display: none !important;
  }
}
```

**Comment by rajgoel** [comment]
Fixed in latest release 4.2.0: https://cdn.jsdelivr.net/npm/reveal.js-plugins@4.2.0/customcontrols/style.css
